src/Host/Broker/Impl/Microsoft.R.Host.Broker.Config.json

```diff
@@ -0,0 +1,3 @@
+﻿{
+  "server.urls": "https://0.0.0.0:5000"
```

Comment:
Now that we're actually configuring the port out of the box, instead of letting the user do so, let's pick something better - 5000 is the default for all ASP.NET Core apps. 

https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.txt